docs/data-management/metrics/ratio-metric.md

```diff
@@ -3,15 +3,54 @@ sidebar_position: 2
+randomization typically leads to overconfident results (underestimating the variance). A proper estimation of the variance must recognize the fact that the observations (orders) are clustered
+at the user level. Although there are several valid approaches for estimating this variance, including [clustered standard errors](https://en.wikipedia.org/wiki/Clustered_standard_errors),
+Eppo uses the approach described in [Deng et al.](https://alexdeng.github.io/public/files/kdd2018-dm.pdf) and recommended in 
+Chapter 18 of _Trustworthy Online Controlled Experiments: A Practical Guide to A/B Testing_ by Ron Kohavi.
 
-Ratio metrics allow you to calculate ratio of two [simple metrics](/data-management/metrics/simple-metric), providing deeper insights into the relative performance of variations. This enables a more nuanced analysis of experimental results, allowing businesses to understand the impact of changes in a more comprehensive manner.
 
-For example, consider an _average order value_ metric, which is created by dividing revenue (sum of prices) by number of orders (_sum_ of items purchased or _count_ of prices).
+The approach is to redefine the metric of interest as a ratio of two simple metrics. For example, take _average order value_, which is defined by 
+dividing revenue (sum of prices) by the number of orders (sum of items purchased or count of prices).
+
 
 $$
 \text{Average order value} = \frac{\text{Revenue}}{\text{Number of orders}}
 $$
 
+We can multiply both the numerator and the denominator by the number of _users_ in the experiment:
+
+$$
+\text{Average order value} = \frac{\text{(Revenue)(Number of users)}}{\text{(Number of orders)(Number of users)}}
+$$
+
+which we can rearrange as follows:
+$$
+\text{Average order value} = \bigg(\frac{\text{Revenue}} {\text{Number of users}}\bigg) \bigg(\frac{\text{Number of users}} {\text{Number of orders}}\bigg)
+$$
+
+which can be expressed as the ratio of two simple metrics:
+$$
+\text{Average order value} = \frac{\text{Revenue per user}}{\text{Number of orders per user}}
+$$
+
+Both the numerator and the denominator are simple metrics whose analysis entity matches the randomization entity (a user). In general, the steps to define a ratio metric are
+as follows:
+1. Define the metric you wish to calculate and identify its analysis entity (e.g., an order in _average order value_).
+2. Divide both the numerator and the denominator by the _randomization_ entity (often a user).
+3. Identify the numerator metric and the denominator metric. The numerator metric will typically be a fact quantity per randomization entity (e.g., revenue per user). The denominator metric will typically be a count of the analysis entity per randomization entity (e.g., number of orders per user).
+4. Make sure that both the numerator metric and the denominator metric are already defined as [simple metrics](/data-management/metrics/simple-metric) in Eppo.
```

Comment:
You shouldn't need to create separate numerator and denominator simple metrics before creating a ratio metric. As long as the fact is defined, you can go straight to creating a ratio metric